cfme/tests/integration/test_external_auth.py

```diff
@@ -55,3 +55,31 @@ def test_appliance_console_ipa_ntp(request, appliance, freeipa_provider):
     # we expect configuration to fail because ntpd is off on freeipa03
     with pytest.raises((Timeout, AssertionError)):
         appliance.configure_freeipa(freeipa_provider)
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.meta(coverage=[1854839])
+def test_chargeback_report():
```

Comment:
Why place this with the external auth test cases, when there's no external authentication involved in the test?

I think this test also belongs in `test_access_control` module with the other test, or in `cfme/tests/intelligence/test_chargeback_manual.py`